### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by piero-vic**
# Feature 1
First pull request for Hello Microverse Project.
## Changes
- Add test.md to .gitignore
- Add style.css
- Add index.html
- Setup README.md
- Setup .gitignore
- Add linters

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
```
Comment (DanteAlonsoHT) [inline .gitignore:2]: - [ ] Instead of "node_modules/" you should type "node_modules".

File: README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> My First Microverse Project.
```
Comment (DanteAlonsoHT) [inline README.md:5]: - [ ] [OPTIONAL] For the next projects you should add a better description, just try to explain what you did, what is the main scope, what kind of technologies or techniques you used, and so on. (Add like 3 lines or more at least in your description in your README file for the next time 😉).

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫♻️ </h1>

Hi @piero-vic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️

- [x] Please, use the [Template provided by Microverse](https://github.com/microverseinc/readme-template) and fill it as you required with your contact information and the info from your repository. For this project, you should remove the "Run tests" and the "Deployment" sections.
- [ ] Fix the linters, you have errors and you should fix your HTML or CSS code according to the linters errors.

_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Instead of `feature-1` you can use a more descriptive feature branch name for the next time such as `created_sidebar`, `added_comments_section`, it is going to depend on what you did.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Always make a professional README file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (piero-vic) [reply README.md]: Hi. I've edited the README.md. Some of the sections on the template seem unnecessary if we take into consideration that this is a small project. Let me know if anything else must be changed.

Comment (DanteAlonsoHT) [reply README.md]: Sure! You can remove the unnecessary sections if you want only for this project 😉

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Piero Lescano Castillo, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**